examples/bridge_echo/cli/src/main.rs

```diff
@@ -21,6 +21,19 @@ async fn main() -> Result<()> {
     let (tx, rx) = unbounded::<Effect>();
     let arc_tx = Arc::new(tx);
 
+    let payload: Vec<_> = (1..10)
+        .map(|id| DataPoint {
+            value: 3.0 * f64::from(id),
+            id: 17,
```

Comment:
Totally inconsequential, but wondered if you meant this to be id rather than 17 each time? ;-)